docs/docsite/rst/community/create_pr_quick_start.rst

```diff
@@ -155,6 +155,8 @@ See :ref:`module_contribution` for some general guidelines about Ansible module
 Test your changes
 =================
 
+	If using the ``docker`` CLI program, the host must be configured to use cgroupsv1 (this is not required for ``podman``).
```

Comment:
If this statement is being added, instructions on how to make the change should be included.

However, I'm currently working on changes to ansible-test that will remove this limitation -- so it may not be worthwhile to update the docs here, given the change would no longer be correct.

Reply:
Thanks @mattclay, I've added the kernel boot argument that's required (but how to update GRUB I've left off as it's distro and system specific).

Regarding the refactor, is it still useful to have this on older versions of the docs? If this is not useful, then no problems if it's closed. Cheers.